system/greenhouse-ccloud/templates/_helper.tpl

```diff
@@ -0,0 +1,29 @@
+{{/* Get related Thanos store endpoints */}}
+{{- define "getRelatedStoreEndpoints" -}}
+{{- $name := index . 0 -}}
+{{- $root := index . 1 -}}
+{{- $stores := list }}
+{{- range $plugin := $root.Values.ingressPlugins -}}
+{{- $thanosStore := printf "thanos-grpc.%s:443" (trimSuffix "." (trimPrefix "ingress." $plugin.recordName)) -}}
+{{- if contains $name $thanosStore -}}
+{{- $stores = append $stores $thanosStore -}}
+{{- end -}}
+{{- end -}}
+{{ range $k, $v := $stores }}
+  - {{ $v }}
+{{- end }}
+{{- end -}}
+
+{{/* Get related Thanos store endpoints */}}
+{{- define "thanosStoreEndpoints" -}}
+{{- range $cluster := .Values.thanos.globalClusters -}}
+{{- $stores := (include "getRelatedStoreEndpoints" (list $cluster.prefix $)) }}
```

Comment:
we could add the range on the prefix for the future already

Reply:
Ok, I did some changes to support it. https://github.com/sapcc/helm-charts/pull/8349/commits/ea604ad96eaf3c36020ac53dbb886362ae3086c7